src/nupic/algorithms/KNNClassifier.py

```diff
@@ -356,7 +356,9 @@ def learn(self, inputPattern, inputCategory, partitionId=None, isSparse=0,
     @param inputCategory (int) The category to be associated to the training
         pattern
 
-    @param partitionId (int) UNKNOWN
+    @param partitionId (int) The k-fold cross validation partition for this
```

Comment:
Why specific to k-folds? It seems this is just an example use for `partitionId`.